widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -7222,196 +7250,312 @@ span.gwt-RadioButton label {
 
 /* ---------- PIR icons ----------  */
 .tundra .dijitEditorIcon.dijitEditorIconUndo {
-  background: url('images/undo.png');
-  width: 22px;
-  height: 22px;
+  --icon-height: 22;
+  --icon-width: 22;
+
+  height: calc(1px * var(--icon-height));
+  width: calc(1px * var(--icon-width));
+
+  background-image: url('images/undo.png');
+  background-position: initial;
 }
 
 .tundra .dijitDisabled .dijitEditorIconUndo {
-  background: url('images/undo_disable.png');
-  width: 22px;
-  height: 22px;
+  --icon-height: 22;
+  --icon-width: 22;
+
+  height: calc(1px * var(--icon-height));
+  width: calc(1px * var(--icon-width));
+
+  background-image: url('images/undo_disable.png');
+  background-position: initial;
 }
 
 .tundra .dijitEditorIcon.dijitEditorIconRedo {
-  background: url('images/redo.png');
-  width: 22px;
-  height: 22px;
+  --icon-height: 22;
+  --icon-width: 22;
+
+  height: calc(1px * var(--icon-height));
+  width: calc(1px * var(--icon-width));
+
+  background-image: url('images/redo.png');
+  background-position: initial;
 }
 
 .tundra .dijitDisabled .dijitEditorIconRedo {
-  background: url('images/redo-disabled.png');
-  width: 22px;
-  height: 22px;
+  --icon-height: 22;
+  --icon-width: 22;
+
+  height: calc(1px * var(--icon-height));
+  width: calc(1px * var(--icon-width));
+
+  background-image: url('images/redo-disabled.png');
+  background-position: initial;
 }
 
 .tundra .dijitEditorIconIndent {
+  --icon-height: 22;
+  --icon-width: 22;
+
+  height: calc(1px * var(--icon-height));
+  width: calc(1px * var(--icon-width));
+
```

Comment:
similar to what was done with other icon styles, at least the `vars` and `height/width` could be aggregated in one selector. eg: `.tundra :is(.dijitEditorIcon.dijitEditorIconUndo, ..., ...) {`

also take a look at the following icon styles (bellow), there is a lot of repetition that can be avoided if we aggregate them 💪🏼